client/README.md

```diff
@@ -2,6 +2,10 @@
 
 etcd/client is the Go client library for etcd.
 
+etcd uses `vendor` directory to manage external dependencies. If `client` is imported
+outside of etcd, the project needs its own vendoring to prevent package conflicts. For more
```

Comment:
Simply copy client to `vendor` dir in your application or use tools like goDep to manage your dependency.